app/Http/Controllers/LoginController.php

```diff
@@ -12,11 +12,10 @@ public function __construct(private LoginRepository $loginRepository)
 
     public function login(LoginRequest $request)
     {
+        $errors = ["Login Credentials are invalid"];
```

Comment:
avoid temporary variables please, if you wont use a variable in more than one place, don't create it. 

Unless it adds meaning to a complex logic which in this case doesn't.

`->withErrors(["Login Credentials are invalid"]);` is better